backend/app/testbot/kb.py

```
@@ -104,34 +104,40 @@ def load_file_index(chatbot_id: str, file_path: str) -> Tuple[Optional[Any], Opt
+
+            index, documents = handle_file_indexing(chatbot_id, file_path)
+            if index is not None and documents is not None:
+                indexes[file_path] = (index, documents)
+
     logger.info(f"Discovered and indexed {len(indexes)} files for {chatbot_id}")
     return indexes
 
+def is_supported_file(file: str) -> bool:
+    return any(file.endswith(ext) for ext in SUPPORTED_EXTENSIONS)
+
+def handle_file_indexing(chatbot_id: str, file_path: str) -> Tuple[Any, List[str]]:
```

Ruff (0.8.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)